docs/operations-guide/migrating-from-h2.md

```diff
@@ -1,29 +1,56 @@
-# Migrating from using the H2 database to Postgres or MySQL/MariaDB
+# Migrating from H2 database to a production database
```

Comment:
Just a note - you changed the title here but the start page of the operations guide still has the former page title ("... to Postgres or MySQL")